docs/spire-helm.md

```diff
@@ -284,21 +277,30 @@ spec:
   - port: 8081
 EOF
 ```
-Once created successfully, set up env. variable:
+
+Once created successfully, set up env. variables based on this service:
+
 ```console
 export SPIRE_SERVER=spire-server.tornjak.svc.cluster.local
 export SPIRE_PORT=8081
 ```
-Assuming the SPIRE server can now be access from `spire` namespace, either via Ingress or Service on port 8081, we can deploy the helm charts.
+
+Assuming the SPIRE server can now be accessed from the `spire` namespace,
+either via Ingress or Service on port 8081,
+we can deploy the helm charts.
 
 We continue using the same cluster name “minikube”,
 trust domain “openshift.space-x.com”
 and region "us-east".
 
-`--debug` flag shows additional information about the helm deployment:
+Use `--debug` flag to show additional information about the helm deployment.
 
 ```console
-helm install --set "spireAddress=$SPIRE_SERVER" --set "spirePort=$SPIRE_PORT"  --set "namespace=$AGENT_NS" --set "clustername=$CLUSTERNAME" --set "region=us-east" --set "trustdomain=openshift.space-x.com" spire charts/spire --debug
+helm install --set "spireAddress=$SPIRE_SERVER" \
+--set "spirePort=$SPIRE_PORT"  --set "namespace=$AGENT_NS" \
+--set "clustername=$CLUSTERNAME" --set "region=us-east" \
+--set "trustdomain=openshift.space-x.com" \
```

Comment:
maybe explain what the `trustdomain` values is used for, and why is this one used (I see a default into the values.yaml)